src/main/resources/META-INF/services/org.apache.lucene.codecs.Codec

```diff
@@ -1,4 +1 @@
-org.opensearch.knn.index.codec.KNN80Codec.KNN80Codec
-org.opensearch.knn.index.codec.KNN84Codec.KNN84Codec
-org.opensearch.knn.index.codec.KNN86Codec.KNN86Codec
```

Comment:
Are we sure we do not use these codecs in any older opendistro version? If these codecs were used to create segments in some older version then assuming they upgrade to latest OpenSearch version, it might fail as they do not find respective codec for reading.

Reply:
Good catch. I think this could have problems for SPI. 

Let me add back these Codecs back.